src/components/NavBar.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const NavBar: React.FC = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'has-background-grey-lighter': isActive },
+            )}
+            to="/"
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'has-background-grey-lighter': isActive },
+            )}
+            to="/people"
+          >
+            People
+          </NavLink>
```

Comment:
Better will be to create our custom component to avoid code duplication. You can rewatch [this video](https://mate.academy/learn/react/react-router?section=video&videoId=3348)